server/app/controllers/admin/AdminApplicationController.java

```diff
@@ -392,21 +381,6 @@ public Result updateStatus(Http.Request request, long programId, long applicatio
     if (maybeRedirectUri.isEmpty()) {
       return badRequest(String.format("The %s field is not present", REDIRECT_URI_KEY));
     }
-    // Verify the UI is changing from the actual current status to detect an out of date UI.
-    if (application.getLatestStatus().isPresent()) {
-      if (!application.getLatestStatus().get().equals(maybeCurrentStatus.get())) {
-        // Only allow relative URLs to ensure that we redirect to the same domain.
-        String redirectUrl = UrlUtils.checkIsRelativeUrl(maybeRedirectUri.orElse(""));
-        return redirect(redirectUrl)
```

Comment:
**question:**  I see that we're now throwing a RuntimeException from the `setStatus` method in this case, rather than this redirect.  Are we handling those RuntimeExceptions somewhere?  How will we notify the user that they need to reload and try again?

Reply:
A lot of these redirects are unnecessary. most of these fields are coming from Http Post form fields and if those values are manipulated, they should fail with a runtime exception. Only case in when a redirect makes sense is catching the "ApplicationAlreadyInStatusException"--> Genuinely Admin can update the status of an application to same status, and we don't want notification to be sent to the applicants. I have added try/catch to catch just this one.